mcpjam-web/src/index.css

```
@@ -0,0 +1,68 @@
+:root {
+  font-family: system-ui, Avenir, Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
+
+  color-scheme: light dark;
+  color: rgba(255, 255, 255, 0.87);
```

Stylelint (17.2.0)

[error] Expected "0.87" to be "87%" (alpha-value-notation)

(alpha-value-notation)